mxnet/mxnet-engine/src/main/java/ai/djl/mxnet/engine/MxSymbolBlock.java

```diff
@@ -253,6 +253,10 @@ public Shape getParameterShape(String name, Shape[] inputShapes) {
     @Override
     public void saveParameters(DataOutputStream os) throws IOException {
         os.writeByte(VERSION);
+        String json = symbol.toJsonString();
```

Comment:
Why not use os.writeUTF()?

Reply:
It's too long and write UTF only has 2 byte length